.github/workflows/next-js-bundle-analysis.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - name: Build Next.js App
         # change this if your site requires a custom build command
-        run: cd .. && pnpm build
+        run: cd .. && export NODE_OPTIONS="--max_old_space_size=4096" && pnpm build
```

Comment:
```suggestion
        run: cd .. && NODE_OPTIONS="--max_old_space_size=4096" pnpm build
```

Reply:
Good call, done in https://github.com/graphprotocol/docs/pull/437/commits/9391487863039f405b77eeba799550c40fd587c3